package.json

```diff
@@ -43,7 +43,7 @@
     "lint-styles": "stylelint '**/*.{css,scss,sass}'",
     "lint-styles:fix": "stylelint '**/*.{css,scss,sass}' --fix",
     "prepublishOnly": "yarn build",
-    "storybook": "BABEL_ENV=storybook NODE_ENV=development start-storybook -p 6006",
+    "storybook": "BABEL_ENV=storybook NODE_ENV=development start-storybook -s ./public -p 6006",
```

Comment:
@juanfabrega I added a `public` folder to host assets like our logo and favicon, but not sure how to make sure it gets publish during our build process. Can you help?